stopcovid/clients/sqs.py

```diff
@@ -45,3 +48,20 @@ def publish_outbound_sms_messages(outbound_sms_messages: List[OutboundSMS]):
     ]
 
     return queue.send_messages(Entries=entries)
+
+
+def publish_drills_to_trigger(
```

Comment:
For discussion: should we clean up this `clients/` pattern in favor of the class-based-with-helper-method pattern we've been using elsewhere?

Reply:
I think we should clean it up. None of it is a huge deal, though I think it makes sense to organize around the actual bounded contexts that have emerged:

- dialog
- drills
- sms/messaging
- initiation/reminders
- status (undecided whether this is part of initiation/reminders)

Reply:
and yes I think I'm using the term properly this time 😄